codeqlAlerts.js

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env node
+"use strict";
+
+require("dotenv").config();
+
+const _ = require("lodash");
+const moment = require("moment");
+const owner = "Sparkpost";
+const Promise = require("bluebird");
+const { Octokit } = require("@octokit/rest");
+const octokit = new Octokit({
+  auth: process.env.GITHUB_TOKEN,
+  // Set GitHub Auth Token in environment variable
+});
+
+const [, , ...args] = process.argv;
+
+//  .number, .created_at, .url, .html_url, .state, .dismissed_by.login, .dismissed_at, .dismissed_reason, .rule.id, .rule.severity, .rule.description, .tool.name, .most_recent_instance.classifications[]]
+async function getCodeAlerts(repos) {
+  return Promise.map(repos, async ({ name, org }) => {
```

Comment:
this would be a good use of `async`, since L24 uses `await`. however, the await isn't needed on L24 as bluebird's `Promise.map` will await the promise on the async function, and even if you were using es6 `Promise.all`, you could merely return the array of promises, rather than awaiting each iterative element.